client/src/components/App.js

```diff
@@ -62,21 +62,23 @@ export default class App extends React.Component {
       timedelay: '',
       refreshDisabled: true,
       delayDisabled: false,
-      minimise: window.navigator.userAgent.toLowerCase().includes('mobi')
+      minimise: window.navigator.userAgent.toLowerCase().includes('mobi'),
+      lang: 'en'
     };
   }
 
   // On loading the page the first message is sent to Dialog Flow with
-  // speech to bring back first message and sending the unique id.  The
-  // next two messages appear through retriggers in the payload
+  // speech to bring back first message and sending the unique id.
   componentDidMount = () => {
     if (!this.state.minimise) {
       this.sendMessage({
-        speech: 'Little window welcome',
-        uniqueId: this.props.uniqueId
+        speech: 'Little Window language selection',
+        uniqueId: this.props.uniqueId,
+        lang: this.state.lang
       });
       this.setState({
-        uniqueId: this.props.uniqueId
+        uniqueId: this.props.uniqueId,
+        lang: this.state.lang
```

Comment:
does `setState` need to set lang here seeing as it is setting it with the lang already in the state?